src/DetailsView/components/settings-panel/settings/bug-filing/bug-filing-settings.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { TextField } from 'office-ui-fabric-react/lib/TextField';
+import * as React from 'react';
+import { FlaggedComponent } from '../../../../../common/components/flagged-component';
+import { FeatureFlags } from '../../../../../common/feature-flags';
+import { NamedSFC } from '../../../../../common/react/named-sfc';
+import { SettingsProps } from '../settings-props';
+
+export const BugFilingSettings = NamedSFC<SettingsProps>('BugFilingSettings', props => {
+    const getBugSettingsUx = () => {
+        return (
```

Comment:
nit: do we need a `return` here? We can use `() => ()` format to eliminate `return` here. Not a blocker though. Just my thought